.changeset/config.json

```diff
@@ -6,7 +6,7 @@
   "access": "public",
   "baseBranch": "main",
   "updateInternalDependencies": "patch",
-  "ignore": ["@example/*", "@test/*"],
+  "ignore": ["!(astro|create-astro|@astrojs/*|astro-scripts)"],
```

Comment:
I switched to using an opt-in syntax so we don't accidentally publish other package names in the future. This pattern is referenced from https://github.com/sveltejs/kit/blob/fb68c253a59402df2b2b1b5db7a54fad504cdd32/.changeset/config.json#L9